Parsers/ASimAuthentication/Tests/Cisco_Meraki_ASimAuthentication_SchemaTest.csv

```diff
@@ -0,0 +1,107 @@
+﻿Result
+"(1) Warning: Missing recommended field [Dst]"
+"(1) Warning: Missing recommended field [DvcAction]"
+"(1) Warning: Missing recommended field [EventSeverity]"
```

Comment:
This should be present based on EventResult.

Reply:
Added where ever possible